src/slurmmanager.jl

```diff
@@ -74,11 +74,67 @@ end
 
 function _new_environment_additions(params_env::Dict{String, String})
   env2 = Dict{String, String}()
+  user_did_specify_JULIA_PROJECT = false
+  user_did_specify_JULIA_LOAD_PATH = false
+  user_did_specify_JULIA_DEPOT_PATH = false
+
   for (name, value) in pairs(params_env)
     # For each key-value mapping in `params[:env]`, we respect that mapping and we pass it
     # to the workers.
     env2[name] = value
```

Comment:
nit: I think this would be slightly clearer if we refactored to use `merge` and `haskey` instead of explicitly looping over keys.

Reply:
I've done some refactoring (https://github.com/kleinhenz/SlurmClusterManager.jl/pull/30/commits/0bde5171b559f90b0fe6bf9f0af97be92a6617fd). I got rid of the loop over key-value pairs. Instead of using `merge()` with an empty Dict, I just did a `copy()`. However, if desired, I'm happy to instead construct an empty Dict and then do a `merge()`.

And I added `haskey()`s for the places where we check if the keys exist.